tools/FakeItEasy.Build/Program.cs

```diff
@@ -1,5 +1,6 @@
 namespace FakeItEasy.Build
 {
+    using System;

```

Comment:
What do you need this for? As far as I can tell, the new code only uses classes from `System.IO`

Reply:
In initial development, I was just spewing messages to the console. I forgot to remove the `using` when I changed that.